Putting openbsd*aarch64 in non-alphabetical order for patch/diff context

Index: configure
--- configure.orig
+++ configure
@@ -7401,6 +7401,9 @@ printf "%s\n" "$PTHREAD_LIBS" >&6; }
 
 openbsd*)
     using_freetype_fontconfig=yes
+    test_kf5=yes
+    test_gtk3_kde5=yes
+    build_skia=yes
     using_x11=yes
     PTHREAD_CFLAGS="-D_THREAD_SAFE"
     PTHREAD_LIBS="-pthread"
@@ -15965,6 +15968,11 @@ openbsd*)
         RTL_ARCH=X86_64
         PLATFORMID=openbsd_x86_64
         ;;
+    aarch64)
+        CPUNAME=AARCH64
+        RTL_ARCH=AARCH64
+        PLATFORMID=openbsd_aarch64
+        ;;
     *)
         as_fn_error $? "Unsupported host_cpu $host_cpu for host_os $host_os" "$LINENO" 5
         ;;
@@ -23101,9 +23109,6 @@ _ACEOF
         rm -fr libconftest*
         { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $gccvisinlineshiddenok" >&5
 printf "%s\n" "$gccvisinlineshiddenok" >&6; }
-        if test "$gccvisinlineshiddenok" = "no"; then
-            as_fn_error $? "Your gcc/clang is not -fvisibility-inlines-hidden safe. This is no longer supported." "$LINENO" 5
-        fi
     fi
 
    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if $CXX_BASE has a visibility bug with class-level attributes (GCC bug 26905)" >&5
@@ -30579,7 +30584,7 @@ then :
   printf %s "(cached) " >&6
 else $as_nop
 
-	for am_cv_pathless_PYTHON in python python2 python3  python3.11 python3.10  python3.9 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3  python3.2 python3.1 python3.0  python2.7 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1  python2.0 none; do
+	for am_cv_pathless_PYTHON in python3  python3.11 python3.10  python3.9 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3  python3.2 python3.1 python3.0  python2.7 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1  python2.0 none; do
 	  test "$am_cv_pathless_PYTHON" = none && break
 	  prog="import sys
 # split strings by '.' and convert to numeric.  Append some zeros
@@ -30977,7 +30982,7 @@ then :
   printf %s "(cached) " >&6
 else $as_nop
 
-	for am_cv_pathless_PYTHON in python python2 python3  python3.11 python3.10  python3.9 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3  python3.2 python3.1 python3.0  python2.7 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1  python2.0 none; do
+	for am_cv_pathless_PYTHON in python3  python3.11 python3.10  python3.9 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3  python3.2 python3.1 python3.0  python2.7 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1  python2.0 none; do
 	  test "$am_cv_pathless_PYTHON" = none && break
 	  prog="import sys
 # split strings by '.' and convert to numeric.  Append some zeros
@@ -31420,7 +31425,7 @@ then :
   printf %s "(cached) " >&6
 else $as_nop
 
-	for am_cv_pathless_PYTHON in python python2 python3  python3.11 python3.10  python3.9 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3  python3.2 python3.1 python3.0  python2.7 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1  python2.0 none; do
+	for am_cv_pathless_PYTHON in python3  python3.11 python3.10  python3.9 python3.8 python3.7 python3.6 python3.5 python3.4 python3.3  python3.2 python3.1 python3.0  python2.7 python2.6 python2.5 python2.4 python2.3 python2.2 python2.1  python2.0 none; do
 	  test "$am_cv_pathless_PYTHON" = none && break
 	  prog="import sys
 # split strings by '.' and convert to numeric.  Append some zeros
@@ -41372,7 +41377,7 @@ printf "%s\n" "internal" >&6; }
     if test "$COM" = "MSC"; then
         HYPHEN_LIB='$(gb_StaticLibrary_WORKDIR)/hyphen.lib'
     else
-        HYPHEN_LIB="-L${WORKDIR}/UnpackedTarball/hyphen/.libs -lhyphen"
+	HYPHEN_LIB="${WORKDIR}/UnpackedTarball/hyphen/.libs/libhyphen.a"
     fi
 fi
 
@@ -41575,7 +41580,7 @@ printf "%s\n" "internal" >&6; }
     if test "$COM" = "MSC"; then
         MYTHES_LIBS='$(gb_StaticLibrary_WORKDIR)/mythes.lib'
     else
-        MYTHES_LIBS="-L${WORKDIR}/UnpackedTarball/mythes/.libs -lmythes-1.2"
+	MYTHES_LIBS="${WORKDIR}/UnpackedTarball/mythes/.libs/libmythes-1.2.a"
     fi
 fi
 
@@ -42626,7 +42631,7 @@ else
     { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking the flex version" >&5
 printf %s "checking the flex version... " >&6; }
     _flex_version=$($FLEX --version | $SED -e 's/^.*\([[:digit:]]\{1,\}\.[[:digit:]]\{1,\}\.[[:digit:]]\{1,\}\).*$/\1/')
-    if test $(echo $_flex_version | $AWK -F. '{printf("%d%03d%03d", $1, $2, $3)}') -lt 2006000; then
+    if test $(echo $_flex_version | $AWK -F. '{printf("%d%03d%03d", $1, $2, $3)}') -lt 2005000; then
         as_fn_error $? "failed ($FLEX $_flex_version found, but need at least 2.6.0)" "$LINENO" 5
     fi
 fi
@@ -47505,7 +47510,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 printf %s "checking for Qt6 libraries... " >&6; }
     qt6_libdir="no"
     for lib_dir in $qt6_libdirs; do
-        if test -r "$lib_dir/$qt6_test_library"; then
+        if ls $lib_dir/$qt6_test_library* /dev/null 2>&1 >/dev/null; then
             qt6_libdir="$lib_dir"
             break
         fi
@@ -47891,7 +47896,7 @@ printf "%s\n" "$kf6_incdir" >&6; }
 printf %s "checking for KF6 libraries... " >&6; }
     kf6_libdir="no"
     for kf6_check in $kf6_libdirs; do
-        if test -r "$kf6_check/$kf6_test_library"; then
+        if ls $kf6_check/$kf6_test_library* /dev/null 2>&1 >/dev/null; then
             kf6_libdir="$kf6_check"
             break
         fi
@@ -48789,16 +48794,7 @@ if test "$USING_X11" = TRUE; then
     else
         XINERAMALIB="$x_libraries"
     fi
-    if test -e "$XINERAMALIB/libXinerama.so" -a -e "$XINERAMALIB/libXinerama.a"; then
-        # we have both versions, let the user decide but use the dynamic one
-        # per default
-        USE_XINERAMA=TRUE
-        if test -z "$with_static_xinerama" -o -n "$with_system_libs"; then
-            XINERAMA_LINK=dynamic
-        else
-            XINERAMA_LINK=static
-        fi
-    elif test -e "$XINERAMALIB/libXinerama.so" -a ! -e "$XINERAMALIB/libXinerama.a"; then
+    if ls $XINERAMALIB/libXinerama.so* /dev/null 2>&1 >/dev/null; then
         # we have only the dynamic version
         USE_XINERAMA=TRUE
         XINERAMA_LINK=dynamic
@@ -50427,7 +50423,7 @@ for i in shasum /usr/local/bin/shasum /usr/sfw/bin/sha
 done
 
 if test -z "$SHA256SUM"; then
-    for i in sha256sum /usr/local/bin/sha256sum /usr/sfw/bin/sha256sum /opt/sfw/bin/sha256sum /opt/local/bin/sha256sum; do
+    for i in gsha256sum /usr/local/bin/sha256sum /usr/sfw/bin/sha256sum /opt/sfw/bin/sha256sum /opt/local/bin/sha256sum; do
         eval "$i --version" > /dev/null 2>&1
         ret=$?
         if test $ret -eq 0; then
